lib/excoveralls/poster.ex

```diff
@@ -22,44 +17,129 @@ defmodule ExCoveralls.Poster do
     end
   end
 
-  defp send_file(file_name, options) do
-    Application.ensure_all_started(:hackney)
+  defp upload_zipped_json(content, options) do
+    Application.ensure_all_started(:ssl)
+    Application.ensure_all_started(:httpc)
```

Comment:
Out of curiosity, why does `:httpc` needs to be started explicitly? Starting `:inets.start()` and `:ssl.start()` typically was sufficient for me in the past 🤔

Reply:
It probably is sufficient, I just overlooked this. PRs welcome 😉

Reply:
Thanks for the comment. Do you see specific negative implication for this extra start logic? (Just a small overhead).

I'll be publishing the release later today (after going back home), but wondering whether to update this before the release.

Reply:
@parroty `httpc` is not an app. Just merge https://github.com/parroty/excoveralls/pull/318 before releasing 😉